pkg/volume/storageos/storageos_util.go

```diff
@@ -305,6 +306,17 @@ func (u *storageosUtil) DeviceDir(b *storageosMounter) string {
 	return ctrl.DeviceDir
 }
 
+// ExpandVolume expands a volume to a new size(in GiB).
+func (u *storageosUtil) ExpandVolume(volumeName, volumeNamespace string, size int) error {
+	opts := storageostypes.VolumeUpdateOptions{
+		Name:      volumeName,
+		Namespace: volumeNamespace,
+		Size:      size,
+	}
+	_, err := u.api.VolumeUpdate(opts)
```

Comment:
Shouldn't we use volume returned by `VolumeUpdate` function to determine size of resized volume?